llvm/docs/LangRef.rst

```diff
@@ -24888,16 +24891,16 @@ Predicated fshr of three vectors of integers.
 Arguments:
 """"""""""
 
-The first three operand and the result have the same vector of integer type. The
-fourth operand is the vector mask and has the same number of elements as the
-result vector type. The fifth operand is the explicit vector length of the
+The first three argument and the result have the same vector of integer type. The
```

Comment:
typo (inherited from the original): `arguments`. There's a few other places where this occurs.